packages/playground/src/cogniteClientPlayground.ts

```diff
@@ -0,0 +1,47 @@
+// Copyright 2020 Cognite AS
+
+import { accessApi, ClientOptions, BaseCogniteClient } from '@cognite/sdk-core';
+import { version } from '../package.json';
+import { DocumentsAPI } from './api/documents/documentsApi';
+
+export default class CogniteClientPlayground extends BaseCogniteClient {
```

Comment:
@andersfylling Why not just `export class CogniteClientPlayground`? You are exporting it anyway with that name in index.ts.

Reply:
I'm unaware why, but this is the same setup as the beta and stable package.